src/diffpy/labpdfproc/tools.py

```diff
@@ -165,3 +166,24 @@ def load_user_metadata(args):
             setattr(args, key, value)
     delattr(args, "user_metadata")
     return args
+
+
+def load_datetime(args):
+    """
+    Load datetime into the provided argparse Namespace.
+
+    Parameters
+    ----------
+    args argparse.Namespace
+        the arguments from the parser
+
+    Returns
+    -------
+    the updated argparse Namespace with datetime inserted as key-value pairs
+    """
+    if args.user_metadata:
```

Comment:
Yes, I get your point. I would follow your instructions to change it. Also, I think using time-stamp is a good idea. Maybe we could change to form of creation_time as you think as a file headers. Additionally, I'm resetting pre-commit since I found out that it did not truly set up after installing it.

Reply:
That's great.  

Our design is to load into args whatever we want in the resulting file-header, then the dump will just dump whatever is in the args into the header.  This is easier to maintain, but it means we really don't want to change the name of a key at dump-time, so I would advocate for calling it `creation_time` also in the args.